src/test/java/org/humancellatlas/ingest/IngestApiClientTest.java

```diff
@@ -85,6 +85,7 @@ public void testGetMetadataDocumentInfo() throws Exception {
         class EnvelopeJson {
             @JsonProperty("uuid") Map<String, Object> uuid;
             @JsonProperty("_links")  Map<String, Object> _links;
+            @JsonProperty("submissionState")  String submissionState;
```

Comment:
What does this do?

Reply:
`submissionState` is a property on the envelope that was missing before and caused tests to fail